src/MigrationBuilder.php

```diff
@@ -28,12 +28,20 @@
 
 final class MigrationBuilder extends AbstractMigrationBuilder
 {
+    /** @psalm-var class-string<ColumnBuilder> */
+    private string|null $columnBuilderClass = null;
+
     public function __construct(
         private ConnectionInterface $db,
         private MigrationInformerInterface $informer,
         private ?int $maxSqlOutputLength = null,
     ) {
-        parent::__construct($this->db->getSchema());
+    }
+
+    /** @psalm-return class-string<ColumnBuilder> */
+    public function columnBuilder(): string
```

Comment:
Hm. `MigrationBuilder` is designed for create columns. May be pass `ConnectionInterface` to [constructor](https://github.com/yiisoft/db-migration/blob/8fb671cd3e0bef1b462fedae4b5f8453596ee994/src/AbstractMigrationBuilder.php#L37) and use it in methods for create columns?